include/onnxruntime/core/common/common.h

```diff
@@ -193,19 +193,16 @@ void LogRuntimeError(uint32_t session_id, const common::Status& status, const ch
                                 ::onnxruntime::MakeString(__VA_ARGS__))
 
 // Check condition. if met, return status.
-#define ORT_RETURN_IF(condition, ...)                                                     \
-  if (condition) {                                                                        \
-    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,                                             \
-                           "Satisfied, but should not be: " #condition "\n",              \
-                           ORT_WHERE.ToString(), ::onnxruntime::MakeString(__VA_ARGS__)); \
+#define ORT_RETURN_IF(condition, ...)                                                                        \
```

Comment:
should we enforce that some error message is provided now?
e.g.: `#define ORT_RETURN_IF(condition, first_part_of_error_message, ...)`

Reply:
oh, i guess it already causes a compile error